### DISCUSSION TITLE
How to build turbo and use that version locally

### DISCUSSION THREAD

**Discussion by fivethreeo** [vercel/turborepo#516] [Help] [answered]
Great if there was some dev docs on this

**Comment by markjm** [comment, marked as answer]
You can set `TURBO_BINARY_PATH` to be "./../../cli/turbo" or whatever your path to your turbo executable is. I have this PR open which would make this easier for the examples in the repo, but there is one issue I need to go back and figure out regarding the different uses of this variable in different places
#518
